ebodac/src/test/resources/enrollBoostVaccination.csv

```diff
@@ -5,4 +5,11 @@ B05-SL10001,3,1968-12-29,M,1,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115
 B05-SL10001,4,1968-12-29,M,1,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115-09-25
 B05-SL10001,5,1968-12-29,M,1,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115-09-26
 B05-SL10001,6,1968-12-29,M,1,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115-09-27
-B05-SL10001,7,1968-12-29,M,1,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115-09-28
\ No newline at end of file
+B05-SL10001,7,1968-12-29,M,1,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115-09-28
+B05-SL10001,11,1968-12-29,M,2,2015-06-30,2015-06-30,,,Boost Vaccination Day,,2115-09-22
```

Comment:
you did not created subjects with required data for the records that you added in this file